python/simulation/fake_raft.py

```diff
@@ -0,0 +1,637 @@
+        hdu.header['CRPIX2A'] = crpix2a
+        hdu.header['WCSNAMEC'] = wcsnamec
+        hdu.header['CTYPE1C'] = ctype1c
+        hdu.header['CTYPE2C'] = ctype2c
+        hdu.header['CRVAL1C'] = crval1c
+        hdu.header['CRVAL2C'] = crval2c
+        hdu.header['PC1_1C'] = pc1_1c
+        hdu.header['PC1_2C'] = pc1_2c
+        hdu.header['PC2_1C'] = pc2_1c
+        hdu.header['PC2_2C'] = pc2_2c
+        hdu.header['CDELT1C'] = 1
+        hdu.header['CDELT2C'] = 1
+        hdu.header['CRPIX1C'] = crpix1c
+        hdu.header['CRPIX2C'] = crpix2c
+        hdu.header['WCSNAMER'] = wcsnamer
+        hdu.header['CTYPE1R'] = ctype1r
+        hdu.header['CTYPE2R'] = ctype2r
+        hdu.header['CRVAL1R'] = crval1r
+        hdu.header['CRVAL2R'] = crval2r
+        hdu.header['PC1_1R'] = pc1_1r
+        hdu.header['PC1_2R'] = pc1_2r
+        hdu.header['PC2_1R'] = pc2_1r
+        hdu.header['PC2_2R'] = pc2_2r
+        hdu.header['CDELT1R'] = cdelt1r
+        hdu.header['CDELT2R'] = cdelt2r
+        hdu.header['CRPIX1R'] = crpix1r
+        hdu.header['CRPIX2R'] = crpix2r
+        hdu.header['WCSNAMEF'] = wcsnamef
+        hdu.header['WCSNAMEB'] = wcsnameb
+        hdu.header['CTYPE1B'] = ctype1b
+        hdu.header['CTYPE2B'] = ctype2b
+        hdu.header['CRVAL1B'] = crval1b
+        hdu.header['CRVAL2B'] = crval2b
+        hdu.header['PC1_1B'] = pc1_1b
+        hdu.header['PC1_2B'] = pc1_2b
+        hdu.header['PC2_1B'] = pc2_1b
+        hdu.header['PC2_2B'] = pc2_2b
+        hdu.header['CDELT1B'] = cdelt1b
+        hdu.header['CDELT2B'] = cdelt2b
+        hdu.header['CRPIX1B'] = crpix1b
+        hdu.header['CRPIX2B'] = crpix2b
+        hdu.header['WCSNAMEQ'] = wcsnameq
+        hdu.header['CTYPE1Q'] = ctype1q
+        hdu.header['CTYPE2Q'] = ctype2q
+        hdu.header['CRVAL1Q'] = crval1q
+        hdu.header['CRVAL2Q'] = crval2q
+        hdu.header['PC1_1Q'] = pc1_1q
+        hdu.header['PC1_2Q'] = pc1_2q
+        hdu.header['PC2_1Q'] = pc2_1q
+        hdu.header['PC2_2Q'] = pc2_2q
+        hdu.header['CDELT1Q'] = cdelt1q
+        hdu.header['CDELT2Q'] = cdelt2q
+        hdu.header['CRPIX1Q'] = crpix1q
+        hdu.header['CRPIX2Q'] = crpix2q
+
+    def write_sensor_image(self, single_sensor_file, slot_name, sensor_id, **kwargs):
+        """
+        Write a FITS image with pixel coordinates matching the specified
+        sensor_id and raft_id.  The output file name is constructed from
+        the test type, sensor type, sensor_id and raft_id.
+
+        Parameters
+        ----------
+        single_sensor_file : str
+            Name of the file to be copied
+        slot_name:  str
+            Name of the slot this sensor occupies
+        sensor_id:  str
+            Name of the sensor, e.g., 'E2V-CCD250-104'
+
+        Keyword arguments
+        -----------
+        raft_id : str
+            Override the raft id
+        process_name_out : str
+            The name of the output eTraveler process, if it differs from process_name
```

Comment:
I think that Eric used **kwargs because pylint complained about the number of arguments being passed.  It may be better overall to just pass the keyword arguments explicitly.